modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -185,6 +185,104 @@ definition {
 			actionName = "Action Name");
 	}
 
+	@description = "LPS-171793 - Verify it's possible to create a notification action with the trigger On Order Status Update"
+	@priority = "5"
+	test CanCreateActionNotificationWithOnOrderStatusUpdate {
+		property portal.acceptance = "true";
+
+		task ("Given: A notification template is created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test test",
+				name = "Notification Template",
+				subject = "Subject");
+
+			Button.clickSave();
+		}
+
+		task ("When: A notification action with the trigger On Order Status Update is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Commerce Order");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Notification action when order status update",
+				active = "true",
+				inputExpression = "orderStatus == 0",
+				notificationName = "Notification Template",
+				thenAction = "Notification",
+				whenAction = "On Order Status Update");
+		}
+
+		task ("And when: an account, minium site and an order is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Accounts Name",
+				accountEntryType = "Business");
+
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "1");
+
+			Refresh();
+
+			CommerceCheckout.gotoCheckout();
+
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "1",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
+
+		task ("And when: The order status is updated") {
+			CommerceNavigator.gotoPortlet(
+				category = "Order Management",
+				portlet = "Orders");
+
+			CommerceNavigator.gotoOrderEntryViaAdmin(status = "Pending");
+
+			Click(
+				key_text = "Accept Order",
+				locator1 = "Link#ANY");
+
+			CommerceOrders.createShipment(
+				complement = "Deliver",
+				moveItemQuantitiesInShipment = "true",
+				moveOrderItemsInShipment = "true",
+				productSkuList = "MIN55861",
+				warehouseName = "Italy");
+		}
+
+		task ("Then: The action is triggered and the notification is sent") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Queue");
+
+			AssertElementPresent(
+				key_objectName = "CommerceOrder",
+				locator1 = "ObjectNotifications#OBJECT_QUEUE_NAME");
```

Comment:
Have you tried testing this with a mock email server running? We should have coverage to see if the email is being sent but that can probably be covered in ObjectNotfications.testcase. This assertion should be fine

Reply:
> Have you tried testing this with a mock email server running? We should have coverage to see if the email is being sent but that can probably be covered in ObjectNotfications.testcase. This assertion should be fine

Not yet, for this case, we just need to test if the notification was sent. Thanks!